package.json

```
@@ -72,7 +72,7 @@
     "rollup-plugin-copy": "^3.1.0",
     "rollup-plugin-hash": "^1.3.0",
     "rollup-plugin-json": "^4.0.0",
-    "rollup-plugin-node-builtins": "^2.1.2",
+    "rollup-plugin-node-builtins": "git+https://github.com/torch2424/rollup-plugin-node-builtins.git",
```

Can we have this in the wasmerio org?

---

Sure! Let's do that adfter we get a working fix for @corwin-of-amber 😄

---

Actually, I just did!

```suggestion
    "rollup-plugin-node-builtins": "git+https://github.com/wasmerio/rollup-plugin-node-builtins.git",
```